### PR TITLE
Weak Hausdorff property

### PR DESCRIPTION
**Pull request by prabau**
Introducing weak Hausdorff (P143) as it is tied to some of the "compactly generated" variants and will be needed in related theorems.

Weak Hausdorff is clearly between T2 and T1.  And it is implied by KC (P100)  Does it imply US (P99) maybe?

### CODE REVIEW

**Comment by StevenClontz** [comment]
(FYI: the default branch for pi-base/data is now `main` to match other repos and the modern convention. But temporarily (because the fix is harder than we anticipated today) the default branch being shown to the public at https://topology.pi-base.org/ is still `master`, so I'll be manually keeping master synced up with main until that's fixed.)

File: theorems/T000227.md [hunk 1]
```diff
@@ -3,10 +3,10 @@ uid: T000227
...
+  P000143: true
 refs:
-- mr: MR2219327
-  name: Minimal properties between T1 and T2.
+- wikipedia: Weak_Hausdorff_space
+  name: Weak Hausdorff space on Wikipedia
 ---
 
-Asserted on page 494 of {{mr:MR2219327}}.
+Evident from the definitions.
```
Comment (StevenClontz) [inline theorems/T000227.md:12]: Maybe

> Continuous images of compact are always compact, and thus closed given {P100}.

**Review by StevenClontz — APPROVED** [review]
Mergable as-is. See my comment to consider elaborating one quick proof.

If you want to add traits to get US-notWH and WH-notKC examples included here let me know and I'll re-review.

File: theorems/T000227.md (hunk 1 above)
Comment (prabau) [reply theorems/T000227.md:12]: I'll add that tomorrow.

**Comment by prabau** [comment]
Revised T227 as suggested.  You can merge at your convenience.  I'll add the examples in a separate PR.

**Pull request merged**